lib/rhc/commands/deployment.rb

```diff
@@ -74,7 +74,9 @@ def activate(id)
 
       raise RHC::DeploymentsNotSupportedException.new if !rest_app.supports? "DEPLOY"
 
-      activate_deployment(rest_app, id)
+      ssh_executable = check_ssh_executable! options.ssh
+
+      activate_deployment(rest_app, id, ssh_executable, !!options.ssh)
```

Comment:
For that matter, maybe it would be cleaner to move the `options.ssh && check_ssh_executable! options.ssh` logic into `activate_deployment` itself (I think it has access to `options` as I describe in a comment below).  That would keep all the SSH-related logic together.